src/Microphysics/saturation_adjustment.jl

```diff
@@ -36,18 +36,19 @@ end
 """
     $(TYPEDSIGNATURES)
 
-Return `SaturationAdjustment` microphysics representing an instantaneous adjustment to
-`equilibrium` between condensates and water vapor, computed by a solver with `tolerance` and `maxiter`.
-
-The options for `equilibrium` are
-    * [`WarmPhaseEquilibrium`](@ref) representing an equilibrium between water vapor and liquid water.
-
-    * [`MixedPhaseEquilibrium`](@ref) representing a temperature-dependent equilibrium between
-      water vapor, possibly supercooled liquid water, and ice. The equilibrium state is modeled as a linear
-      variation of the equilibrium liquid fraction with temperature, between
-      the freezing temperature (e.g. 273.15 K) below which liquid water is supercooled,
-      and the temperature of homogeneous ice nucleation temperature (e.g. 233.15 K) at which
-      the supercooled liquid fraction vanishes.
+Return `SaturationAdjustment` microphysics representing an instantaneous adjustment
+to `equilibrium` between condensates and water vapor, computed by a solver with
+`tolerance` and `maxiter`.
+
+The options for `equilibrium` are:
+* [`WarmPhaseEquilibrium`](@ref) representing an equilibrium between water vapor and liquid water.
+
+* [`MixedPhaseEquilibrium`](@ref) representing a temperature-dependent equilibrium between
+    water vapor, possibly supercooled liquid water, and ice. The equilibrium state is modeled as a linear
+    variation of the equilibrium liquid fraction with temperature, between
+    the freezing temperature (e.g. 273.15 K) below which liquid water is supercooled,
+    and the temperature of homogeneous ice nucleation temperature (e.g. 233.15 K) at which
+    the supercooled liquid fraction vanishes.
```

Comment:
@glwagner, I know in plain text adding 4-space tabs making the text read better but then markdown interprets it differently and messes things up. See [how this docstring appears on `main` for example](https://numericalearth.github.io/BreezeDocumentation/dev/api/#Breeze.Microphysics.SaturationAdjustment).

Reply:
Yes! Or doesn't even do the line breaks for the itemized list.